starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -201,7 +201,8 @@ sub set_ACA_bad_pixels {
 	#cut out the quadrant boundaries
 	foreach my $i ($line[0]..$line[1]) {
 	    foreach my $j ($line[2]..$line[3]) {
-		my $pixel = {};
+		my $pixel = {'row' => $i,
+                             'col' => $j};
```

Comment:
This is not strictly related to dither, but helps with an improved warning message for bad pixels.